### PR TITLE
Update getting started information.

### LINKED ISSUE #348: Update Getting Started guide on website
Out _Getting Started_ guide is hopelessly outdated.

### PR DESCRIPTION
**Pull request by soerendomroes**
Fixes #348

@Eddykasp

### CODE REVIEW

File: docs/content/gettingstarted/_index.md [hunk 1]
```diff
@@ -6,74 +6,25 @@ menu:
     weight: 20
 ---
 
-Automatic layout can be a complex thing, but starting to use the _Eclipse Layout Kernel_ can be as easy as installing a few features from our update site.
+Automatic layout can be a complex thing, but starting to use the _Eclipse Layout Kernel_ can be as easy.
```
Comment (Eddykasp) [inline docs/content/gettingstarted/_index.md:9]: remove as

File: docs/content/gettingstarted/_index.md [hunk 2]
```diff
@@ -6,74 +6,25 @@ menu:
...
 
-{{% note title="A Quick Warning" mode="danger" %}}
-This guide refers to the pre-0.1.0 version of the Eclipse Layout Kernel. Since then, much has changed. We will try to get this guide updated as soon as possible. In the meantime, you may want to check out the rest of the documentation.
-{{% /note %}}
+First of all, there are two versions of ELK. The [Java Version](https://github.com/eclipse/elk) and the Javascript Version, which is transpiled from the Java source using [GWT](https://www.gwtproject.org/).
 
+Examples of ELK graphs can be found [here](https://rtsys.informatik.uni-kiel.de/elklive/examples.html) in the [elklive](https://rtsys.informatik.uni-kiel.de/elklive/index.html) tool, which supports writing ELK graphs in the [ELKT](https://rtsys.informatik.uni-kiel.de/elklive/elkgraph.html) and [JSON](https://rtsys.informatik.uni-kiel.de/elklive/json.html) format, can convert between different formats, and allows to browse additional models.
```
Comment (Eddykasp) [inline docs/content/gettingstarted/_index.md:13]: allows browsing additional

File: docs/content/gettingstarted/_index.md [hunk 3]
```diff
@@ -6,74 +6,25 @@ menu:
...
 
-First off, you will have to actually install ELK. Head over to our downloads section and paste the URL of our update site into your Eclipse installation. You will discover four features:
+## Issues and Questions
 
-* The core feature contains the basic layout kernel infrastructure. This one you definitely need to install.
-* Depending on what technology your editor is based on, you may be lucky enough to find that we have already done much of the work necessary to make your editor work with ELK. We provide out-of-the-box interfaces for GMF-based editors, which work well without customization for many editors. Note that prior to release 0.7.0 there has been support for Graphiti editors as well. If you are interested, feel free to check out the old code.
-* While ELK comes with a number of layout algorithms, you may also want to use the [Graphviz](http://www.graphviz.org/) library. Installing the corresponding feature allows ELK to use Graphviz if it is installed on your computer.
+Both ELK and elkjs allow to report bugs and to ask questions in form of GitHub issues.
```
Comment (Eddykasp) [inline docs/content/gettingstarted/_index.md:19]: maybe add link to issue template on github

Comment (soerendomroes) [reply docs/content/gettingstarted/_index.md:19]: Isn't the template currently only added in elkjs? Do you think we need something similar for ELK?

Comment (soerendomroes) [reply docs/content/gettingstarted/_index.md:19]: It is not possible to add templates to ELk, but I am unsure why this is the case.

Comment (Eddykasp) [reply docs/content/gettingstarted/_index.md:19]: we can still link to [elkjs](link to issue template) and [elk](link to create issue)

in principle the they should have the same templates though I think

**Pull request merged**